### ISSUE TITLE
[NPC] Risen Zombie

### ISSUE THREAD

**Issue by Dranah** [WoWManiaUK/Redemption#2517]
These are zombies inside Culling of Stratholme (http://www.wow-mania.com/armory/?npc=27737) related to achievement Zombiefest! http://www.wowhead.com/achievement=1872

How it works right now:
- Zombies have same movement speed as regular elites, making them impossible to be kited
- Zombies have very little aggro distance, you have to walk on right onto them to get their attention

How it should work:
- Zombies should move REALLY slow, making it possible to easily kite them with your normal run speed
- Zombies should easily aggro to you if you run past near them, from like 10-15 yards

https://www.youtube.com/watch?v=leHoVmyQ5Wk >> here's a video showing their intended movement
Tl;DR this achievement is a lot easier on retail than on mania atm

**Comment by fvicaria** [comment]
This has been tested and shown that works. There are some random elements on the number and timers of spawns that cannot be fixed.

**Comment by TheManiax** [comment]
Can you just reduce their movement speed to something experimentally that allows the player to kite them?

**Comment by Dranah** [comment]
zombies are now slower and kitable
